Button 2: More Block padding

swoodipto opened the pull request:
#### Changes proposed in this Pull Request:

Wrap `more-link` _(added from Block Editor)_ inside a paragraph so that **"Continue Reading →"** does not shift below the post divider.

#### Related issue(s):

#2129 

#### BEFORE:

![#2129-before-site](https://user-images.githubusercontent.com/29238672/89543669-d442fd80-d81e-11ea-824e-7ba2f67ca51b.png)
![#2129-before-html](https://user-images.githubusercontent.com/29238672/89543675-d60cc100-d81e-11ea-92cb-99cc2e46edc5.png)

#### AFTER:

![#2129-after-site](https://user-images.githubusercontent.com/29238672/89544278-8bd80f80-d81f-11ea-8eb7-a3f415bb1202.png)
![#2129-after-html](https://user-images.githubusercontent.com/29238672/89544283-8da1d300-d81f-11ea-8c06-aee298fb400d.png)

swoodipto committed:
#2129: Button 2: More Block padding

pbking reviewed (dismissed):
I believe this could be done with a css-only solution without introducing additional functions or wrapping the element in a `<p>` tag.

The `<a .more-link>` seems that it shouldn't be absolute positioned and eliminating the forced height and padding-top positions the element where it should be.

Therefore I suggest instead that we delete CSS rather than adding PHP to address.

pbking committed:
Eliminate unnecessary padding in 'More' link

Removed initial change to wrap more link in a `<p>` tag.
Change 'more' link position from 'absolute' so that it would lay out as
expected.
Removed the 'more link' paddings and unnecessary size to allow the flow
to layout as expected.

pbking commented:
Made the proposed changes and requesting review by another.

MaggieCabrera reviewed (approved):
LGTM!

MaggieCabrera committed:
bumped version

MaggieCabrera commented:
D52718-code
